### ISSUE TITLE
Client cleanup

### ISSUE THREAD

**Issue by spicman** [MrMatynator/MatyKalMapEditor#18]
It would be good to have the possibility of checking the OPL to Objects folders and giving an option to clean up (move unused to a backup folder structure similar to what is currently in Objects folder to know where those items were removed from).

**Comment by leegmiller** [comment]
Yes i second this, I really would love this feature the objects folder gets bigger and bigger but 50% of them are probably not even used, someway to weed out without doing it manually would be a great benefit of this editor.
